repair/row_level.cc

```diff
@@ -1118,6 +1146,63 @@ class repair_meta {
             // We are about to create a real evictable reader, so drop the fake
             // reader (evicted or not), we don't need it anymore.
             _db.local().get_reader_concurrency_semaphore().unregister_inactive_read(std::move(_fake_inactive_read_handle));
+            if (is_incremental_repair()) {
```

Comment:
We need to improve the structure of the code here, this method is already > 100 lines and now we add another 100. Nobody can follow this code like this.

Reply:
Fixed.